edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc

```diff
@@ -6,44 +6,53 @@ include::_attributes/common-attributes.adoc[]
-
 // Lifecycle Agent (LCA)
 
-include::modules/ztp-image-based-upgrade-prep.adoc[leveloffset=+1]
+You can use a single resource on the hub cluster, the `ImageBasedGroupUpgrade` custom resource (CR), to manage an imaged-based upgrade on a selected group of managed clusters through all stages.
+{cgu-operator-first} reconciles the `ImageBasedGroupUpgrade` CR and creates the underlying resources to complete the defined stage transitions, either in a manually controlled or a fully automated upgrade flow.
+
+For more information about the image-based upgrade, see "Understanding the image-based upgrade for single-node OpenShift clusters".
 
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../edge_computing/ztp-preparing-the-hub-cluster.adoc#ztp-preparing-the-ztp-git-repository-ver-ind_ztp-preparing-the-hub-cluster[Preparing the {ztp} site configuration repository for version independence]
+* xref:../../edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.adoc#cnf-understanding-image-based-upgrade[Understanding the image-based upgrade for single-node OpenShift clusters]
 
-* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.adoc#ztp-image-based-upgrade-prep-resources[Creating ConfigMap objects for the image-based upgrade with {lcao} using {ztp}]
+//include::modules/ztp-image-based-upgrade-concept.adoc[leveloffset=+1]
+
+include::modules/ztp-image-based-upgrade-procedure-steps.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
 
 * xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.adoc#ztp-image-based-upgrade-shared-container-partition_shared-container-partition[Configuring a shared container partition between ostree stateroots when using {ztp}]
 
+* xref:../../edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.adoc#ztp-image-based-upgrade-prep-resources[Creating ConfigMap objects for the image-based upgrade with {lcao} using {ztp}]
+
 * xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc#oadp-about-backup-snapshot-locations_installing-oadp-ocs[About backup and snapshot locations and their secrets]
 
 * xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR]
 
 * xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[Creating a Restore CR]
 
-include::modules/ztp-image-based-upgrade-upgrade.adoc[leveloffset=+1]
+//* xref:../../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-supported-combinations_ztp-gitops[Supported action combinations]
```

Comment:
When will this link be commented in?

Reply:
I ask as I see the following statement: 

![Screenshot from 2025-01-09 17-01-25](https://github.com/user-attachments/assets/cf5c71e1-1d35-4f0d-828d-6960d887807c)

Maybe this admonition should be removed?